### PR TITLE
Update Travis token and Repo ID to work with `travis-ci.com`

### PR DESCRIPTION
**Pull request by Eric-Arellano**
Updating the URL used for the Travis API in https://github.com/pantsbuild/rbe-token-server/commit/ef397f576ef5b1905b77dcfed600a014bb9d2a3a was not enough to fix Travis migrating us from travis-ci.org to travis-ci.com.

We also needed to update the repo ID and regenerate the `travis token`.

### Steps to generate the token
1. Ensure that the account `pantsbuild-ci-bot` is hooked up to Travis and GitHub for the repo `pantsbuild/pants`
1. `$ travis login --com --user pantsbuild-ci-bot`
1. `$ travis token --com > token.txt`
1. `$ gcloud kms encrypt --keyring=rbe-token-server --key=travis --location=global --plaintext-file=token.txt --ciphertext-file=encrypted.txt --project=pants-remoting-beta`
1. `$ cat encrypted.txt | base64 | pbcopy`
1. Update `README.md` and `app.yaml` to use the result of the last step.

What's going on here? As explained in https://github.com/pantsbuild/rbe-token-server/commit/e4e6ae2355708f4b54b5b3e3b62086e8c9333e6e, we need a secure way to store the Travis token in this public repository. 

Google App Engine does not have support for secret env vars, so we use Google KMS to secure the token and safely decrypt it in the Google App Engine server. To do this, we use the `travis` key we created in https://github.com/pantsbuild/rbe-token-server/commit/e4e6ae2355708f4b54b5b3e3b62086e8c9333e6e.

However, `gcloud kms encrypt` saves the key as binary and we need ASCII to work properly with `app.yaml`. So, we encrypt the ciphertext with base64.

### CODE REVIEW

File: src/travis.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ def _get_travis_token() -> str:
     )
     token: str = kms_client.decrypt(
         name=resource_name, ciphertext=b64decode(os.environ[env_var])
-    ).plaintext.decode()
+    ).plaintext.decode().strip()
```
Comment (Eric-Arellano) [inline src/travis.py:34]: The decrypted key has `\n` in it. I'm not sure how I avoided that originally - removing the `\n` in `--plaintext-file=token.txt` had no impact. Don't think it matters much, though.

File: src/travis.py [hunk 2]
```diff
@@ -8,8 +8,8 @@
 import requests
 from google.cloud import kms_v1
 
-DATE_MECHANISM_FIRST_ENABLED = datetime(year=2019, month=7, day=15, tzinfo=timezone.utc)
-PANTSBUILD_PANTS_REPO_ID = 402860
+DATE_MECHANISM_FIRST_ENABLED = datetime(year=2020, month=1, day=22, tzinfo=timezone.utc)
```
Comment (Eric-Arellano) [inline src/travis.py:11]: Not strictly necessary, but figure it's slightly safer this way. I chose yesterday so that builds from last evening can work.

**Comment by Eric-Arellano** [comment]
Going to merge to stabilize CI.

**Review by jsirois — COMMENTED** [review]
LGTM

**Pull request merged**